docs/api-types/index-v1alpha1.md

```diff
@@ -332,3 +333,14 @@ TLog specifies the URL to a transparency log that holds the signature and public
 | trustRootRef | Use the Public Key from the referred TrustRoot.TLog | string | false |
 
 [Back to TOC](#table-of-contents)
+
+## URL
+
+URL defines all the propierties to fetch a remote policy
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| url | URL to the policy data. | apis.URL | false |
+| sha256sum | Sha256sum defines the shasum of the policy hosted in the url. | string | false |
```

Comment:
Wonder if we should say it's the exact computation of the 'body' of the http response?

Reply:
yes, i like that.